Challenge_4.rb

```diff
@@ -0,0 +1,12 @@
+number = (1..100).to_a.shuffle.pop
+tries = 0
+
+puts 'guess a number between 1 and 100'
+
+player_guess = gets.chomp
+tries += 1
+
+def guess(number, player_guess, tries)
+  if player_guess.to_i == number.to_i
+    puts'you got it #{number}you got it in #{tries} tries'
+  elsif player_guess.to_i > 100 || player_guess.to_i < 1
```

Comment:
Alternative for you
  number = rand(1..100)
 
  puts'Guess a number between 1 and 100'
  guess = gets.chomp.strip
 
  until guess.to_i == number
 
    if guess.to_i > number
      puts "Your guess is too high, please guess again"
 
    elsif guess.to_i < number
      puts "Your guess is too low, please guess again"
 
    end
    guess = gets.chomp.strip
  end
  guess=-1
  puts "You are correct, the number was '#{number}'!"